actors/builtin/market/market_test.go

```diff
@@ -404,6 +407,36 @@ func TestPublishStorageDeals(t *testing.T) {
 
 		totalStorageFee = big.Sum(totalStorageFee, deal4.TotalStorageFee(), deal5.TotalStorageFee())
 		require.EqualValues(t, totalStorageFee, st.TotalClientStorageFee)
+
+		// PUBLISH DEALS with a different provider
+		owner := tutil.NewIDAddr(t, 108)
+		provider := tutil.NewIDAddr(t, 109)
+		worker := tutil.NewIDAddr(t, 110)
```

Comment:
Only the provider address need to differ, so the test is slightly stronger if limited to that. Yes, it's realistic for providers to share owner/worker.

Reply:
This is done.